src/views/BivariateManager/BivariateManager.module.css

```diff
@@ -4,49 +4,42 @@
   display: flex;
   min-width: 845px;
   overflow-x: auto;
-}
-
-.Nav,
-.List,
-.LegendMap {
-  display: flex;
-  flex-direction: column;
-}
 
-.LegendMap {
-  width: 492px;
-  flex-shrink: 0;
-}
+  & .Nav {
+    display: flex;
+    flex-direction: column;
+    width: 188px;
+    flex-shrink: 0;
+    border-right: 1px solid var(--faint-weak);
+    padding-top: var(--half-unit);
 
-.Nav {
-  width: 188px;
-  flex-shrink: 0;
-}
+    & .Tab {
+      font: var(--font);
+      line-height: 40px;
+      display: flex;
+      width: 100%;
 
-.ListFilters {
-  display: flex;
-  align-items: center;
-  padding: 24px 32px 16px 32px;
-}
+      &[aria-selected='true'] {
```

Comment:
it's kinda of hack?
why not :selected ?

Reply:
**:selected** Do you mean custom pseudo-class? Not sure that they are already implemented yet. Or you mean to use selected prop of component? Right now we dont have **selected** attribute automatically applied to selected **Tab**. Tabs state is controlled through **selectedIndex** prop of general **Tabs** component (not throgh single **Tab** property). But we have "aria-selected" on selected tab as a part of WAI-ARIA reccomendations. So, to style selected tab I used it as selector as well. To apply your suggestion and use **selected** prop we will need more changes in code to implement such behavior

Reply:
Sorry, I mean [:checked](https://developer.mozilla.org/en-US/docs/Web/CSS/:checked) pseudo-class.
A habit from the old school of implementing tabs via radio so as not to recreate control with the keyboard and use the `checked` and `disabled` state out of the box. But I look into Tabs component and now see it have this implementations in react, so now i understand why aria used for styling.